scripts/moods.js

```diff
@@ -38,4 +38,12 @@ getMoodHTML, taking mood
     -close div
     -return returnHTML
 
-*/
\ No newline at end of file
+*/
+const getMoodHTML=(mood)=>{
+    let moodHTML=`<div class="cell">`
+    moodHTML+=`<div class="moodImg">${mood.imageUrl}</div>
+            <div class="moodName">${mood.name}</div>
+            <div class="moodQuotes">${mood.quotes[0]}\n ${mood.quotes[1]}</div>
```

Comment:
please make a div to contain this div(#moodQuotes) and also a paragraph with the string "Macho Advice:"